### DISCUSSION TITLE
Manage DNS Record with Pro Team Account

### DISCUSSION THREAD

**Discussion by hawki2sl** [vercel/community#2307] [Help]
### Summary

Previously when my project was under my hobby account I was able to add and mange my DNS record for my domain. But since I transferred the project to my team account it doesn't seem to give me that option anymore?

**Comment by amyegan** [comment]
Hi @hawki2sl. You should be able to manage DNS records for your Pro team's domains on the Domains tab, much like you would on a Hobby account. Only the domains linked to that team will be shown, so you'll need to move the domain from your hobby account if you want it to be associated with your team. 

You can learn more about the process in this doc: [Transferring Domains to Another Team or Project](https://vercel.com/docs/concepts/projects/domains/transfer-your-domain#)
